src/main/resources/swagger/api-docs.yaml

```diff
@@ -1425,6 +1425,49 @@ paths:
           description: Method Configuration failed to resolve input expressions with the supplied Entity
         500:
           description: Internal Error
+  '/api/libraries':
+    get:
+      tags:
+        - Library
+      operationId: FindDocuments
+      summary: |
+        lists published library datasets
+      produces:
```

Comment:
also don't need produces anymore! (check the readme in src/main/resources/swagger